src/main/resources/template-ls.json

```diff
@@ -0,0 +1,83 @@
+{
+  "languageServers": [
+    {
+      "name": "CSS Language Server",
+      "instructions": [
+        "To consume CSS language server you need to:",
+        "<ul>",
+        "  <li><a href=\"https://nodejs.org/en/download\">download and install nodejs</a></li>",
+        "  <li><a href=\"https://code.visualstudio.com/download\">download and install vscode</a>.</li>",
+        "  <li>Once vscode is installed, it will store the CSS language server in <b>${BASE_DIR}/resources/app/extensions/css-language-features/server/dist/node/cssServerMain.js</b></li>",
+        "  LSP4IJ tries to generate the proper <b>cssServerMain.js</b> file path according to your OS, but you will have to perhaps adjust it.",
+        "  </li> ",
+        "</ul>"
+      ],
+      "runtime": "node",
+      "programArgs": {
+        "windows": "\"C:/Users/${user.name}/AppData/Local/Programs/Microsoft VS Code/resources/app/extensions/css-language-features/server/dist/node/cssServerMain.js\" --stdio",
+        "mac": "\"/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/server/dist/node/cssServerMain.js\" --stdio",
+        "unix": "\"/usr/share/code/resources/app/extensions/css-language-features/server/dist/node/cssServerMain.js\" --stdio",
+        "default": "\resources/app/extensions/css-language-features/server/dist/node/cssServerMain.js\" --stdio"
+      },
+      "fileTypeMappings": [
+        {
+          "fileType": "CSS",
+          "languageId": "css"
+        }
+      ]
+    },
+    {
+      "name": "HTML Language Server",
+      "instructions": [
+        "To consume HTML language server you need to:",

```

Comment:
You can use Visual Studio Code's HTML language server by following these instructions: